tools/pretty-save.js

```diff
@@ -1,19 +1,27 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 const fs = require("fs").promises;
 const path = require("path");
+const { isBinaryFormat } = require("../electron/saveDataBinaryFormat");
 
 async function getSave(file) {
-  const data = await fs.readFile(file, "utf8");
+  const data = await fs.readFile(file);
+
+  let jsonSaveString;
```

Comment:
Why is this duplicating code from the utils?

Reply:
This is a standalone js tool, it's not going to be able to import code from the ts utils files.